create-yum-repo.sh

```diff
@@ -17,3 +18,5 @@ for DIST in $DISTS; do
 
   createrepo $DIST
 done
+
+cd ../..
```

Comment:
It's not strictly necessary as this is run inside Docker. I added it so that if you run the script locally on your machine (outside Docker), it leaves you in the same directory that you started (the top-level repo root). I'll add a comment, but am happy to remove it if it causes some other problem